src/app/shared/model/results/scores/results-container.model.ts

```diff
@@ -25,6 +25,8 @@ import {MaxpoolFusionFunction} from '../fusion/maxpool-fusion-function.model';
 import {MediaObjectMetadataDescriptor, MediaObjectQueryResult, MediaSegmentDescriptor, MediaSegmentQueryResult, StringDoublePair} from '../../../../../../openapi/cineast';
```

Comment:
Multiple things on this diff which Github does not let me comment at the appropriate point:
- you should include the temporal results in the serialize() method. This is important to log the results.
- you should include an update to the observable in the next() method - i don't think this works otherwise? I don't see where the observable gets updated otherwise
- maybe add a comment in the rerank() method that there is no need to update temporal results, since no new information comes in about them.

Reply:
Thanks for the feedback, I tried to implement it in my most recent commit. 

Regarding the serialize method, I am not 100% sure what would be the best way to do this, maybe we could discuss this in the next meeting.